dbt/models/model/schema.yml

```diff
@@ -4,61 +4,192 @@ sources:
+                - run_id
+              config:
+                  error_if: ">5748"
+              meta:
+                description: assessment card should be unique by pin, card, year, and run_id
         tags:
           - load_auto
 
       - name: assessment_pin
         description: '{{ doc("table_assessment_pin") }}'
+        data_tests:
+          - unique_combination_of_columns:
+              name: model_assessment_pin_unique_by_pin_year_and_run_id
+              combination_of_columns:
+                - meta_pin
+                - meta_year
+                - run_id
+              config:
+                  error_if: ">2016"
+              meta:
+                description: assessment pin should be unique by pin, year, and run_id
         tags:
           - load_auto
 
       - name: feature_importance
+        data_tests:
+          - unique_combination_of_columns:
+              name: model_feature_importance_unique
+              combination_of_columns:
+                - year
+                - run_id
+                - model_predictor_all_name
+              meta:
+                description: feature importance should be unique by year, run_id, and model_predictor_all_name
         description: '{{ doc("table_feature_importance") }}'
         tags:
           - load_auto
 
       - name: metadata
+        data_tests:
+          - unique_combination_of_columns:
+              name: model_metadata_unique_by_year_and_run_id
+              combination_of_columns:
+                - year
+                - run_id
+              meta:
+                description: metadata should be unique by year and run_id
         description: '{{ doc("table_metadata") }}'
         tags:
           - load_auto
 
       - name: parameter_final
         description: '{{ doc("table_parameter_final") }}'
+        data_tests:
+          - unique_combination_of_columns:
+              name: model_parameter_final_unique_by_year_and_run_id
+              combination_of_columns:
+                - year
+                - run_id
+              meta:
+                description: parameter final should be unique by year and run_id
         tags:
           - load_auto
 
       - name: parameter_range
         description: '{{ doc("table_parameter_range") }}'
+        data_tests:
+          - unique_combination_of_columns:
+              name: model_parameter_range_unique_by_year_run_id_and_parameter_name
+              combination_of_columns:
+                - year
+                - run_id
+                - parameter_name
+              meta:
+                description: parameter range should be unique by year run_id and parameter_name
         tags:
           - load_auto
 
       - name: parameter_search
         description: '{{ doc("table_parameter_search") }}'
+        data_tests:
+          - unique_combination_of_columns:
+              name: model_parameter_search_unique_by_year_run_id_and_iteration
+              combination_of_columns:
```

Comment:
**issue:** I believe the full primary key here is `year`, `run_id`, `iteration`, `configuration`, `fold_id`. Let's update this test and the docs to reflect that.